px-data-grid.html

```diff
@@ -62,6 +60,9 @@
 
       </div>
     </template>
+
+    <px-data-grid-filters-preview filters="[[_filters]]" columns="[[columns]]" localize="[[_boundedLocalize]]"></px-data-grid-filters-preview>
```

Comment:
nit: in future, split long lines like this one ( > 100 chars) into multiple lines:

```html
<px-data-grid-filters-preview 
  filters="[[_filters]]" 
  columns="[[columns]]" 
  localize="[[_boundedLocalize]]">
</px-data-grid-filters-preview>
```

It's easier to read and to do CR.